README.md

```diff
@@ -390,18 +392,12 @@ Command to stop the continuous data stream of profile messages.
 
 ### set
 
-#### 2000 parameters
-This is the main parameter block.
+#### 2000 device_id
+Change the device id
 
 | Type | Name             | Description                                                      | Units |
 |------|------------------|------------------------------------------------------------------|-------|
-| u8 | mode | Operating mode (1 for Ping360) |  |
-| u8 | gain_setting | Analog gain setting (0 = low, 1 = normal, 2 = high) |  |
-| u16 | angle | Head angle | gradian |
-| u16 | transmit_duration | Acoustic transmission duration (1~1000 microseconds) | microsecond |
-| u16 | interval | Time interval between individual signal intensity samples in 25nsec increments (40 to 40000 == 1 microsecond to 1000 microseconds) |  |
-| u16 | transmit_frequency | Acoustic operating frequency |  |
-| u16 | number_of_samples | Number of samples per reflected signal |  |
+| u16 | id | Device ID (1-254). 0 and 255 are reserved. |  |
```

Comment:
u8 id
u8 reserved
Using u16 does not match with protocol definition.

Reply:
Ok.